AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -111,4 +126,59 @@ module HierarchicalVersionUtils {
     assume {:axiom} fresh(Crypto) && fresh(Crypto.Modifies);
     return Success(Crypto);
   }
+
+  method ValidateMdDigest (
```

Comment:
OK. I think see where you are going;
you are going to replace the duplicated code in `GetKeys` with this method,
yes?

Reply:
yes